### PR TITLE
List all apps

### PR DESCRIPTION
**Pull request by guillemcordoba**
Depends on https://github.com/holochain/holochain/pull/832

### CODE REVIEW

File: src/api/admin.ts [hunk 1]
```diff
@@ -150,6 +150,9 @@ export type ListCellIdsResponse = Array<CellId>
 export type ListActiveAppsRequest = void
 export type ListActiveAppsResponse = Array<InstalledAppId>
 
+export type ListInActiveAppsRequest = void
```
Comment (zippy) [inline src/api/admin.ts]:
```suggestion
export type ListInactiveAppsRequest = void
```

File: src/api/admin.ts [hunk 2]
```diff
@@ -150,6 +150,9 @@ export type ListCellIdsResponse = Array<CellId>
...
+export type ListInActiveAppsResponse = Array<InstalledAppId>
```
Comment (zippy) [inline src/api/admin.ts]:
```suggestion
export type ListInactiveAppsResponse = Array<InstalledAppId>
```

File: src/websocket/admin.ts [hunk 3]
```diff
@@ -68,6 +68,8 @@ export class AdminWebsocket implements Api.AdminApi {
     = this._requester('list_cell_ids')
   listActiveApps: Requester<Api.ListActiveAppsRequest, Api.ListActiveAppsResponse>
     = this._requester('list_active_apps')
+  listInactiveApps: Requester<Api.ListInActiveAppsRequest, Api.ListInActiveAppsResponse>
```
Comment (zippy) [inline src/websocket/admin.ts]:
```suggestion
  listInactiveApps: Requester<Api.ListInactiveAppsRequest, Api.ListInactiveAppsResponse>
```

**Review by zippy — APPROVED** [review]

**Comment by guillemcordoba** [comment]
@zippy I don't like the way the API looks with the weird `admin.listApps({status_filter: {'Active': null}})`, but looks like it's an upstream problem of messagepack-serde (see https://github.com/holochain/holochain/pull/279 and https://github.com/3Hren/msgpack-rust/issues/250). What should be done here? For now everything works, but with the weird API. 

Alternatively, we could accept a generic string in the filter and return an error if it doesn't match 'active' or 'inactive'. Yes I think that's better, let me change to that.

File: src/api/admin.ts [hunk 4]
```diff
@@ -150,7 +150,7 @@ export type ListCellIdsResponse = Array<CellId>
 export type ListActiveAppsRequest = void
 export type ListActiveAppsResponse = Array<InstalledAppId>
 
-export type AppStatusFilter =  {'Active': null }| {'Inactive': null};
+export type AppStatusFilter =  'active' | 'inactive';
```
Comment (Connoropolous) [inline src/api/admin.ts]: Typescript will support nice code preview with this syntax right? No need to make it an enum structure?

**Review by Connoropolous — COMMENTED** [review]
I like that you're hiding the awkward series serialization. I still can't believe they don't have reasonable serialization for flat enums

File: src/api/admin.ts (hunk 4 above)
Comment (guillemcordoba) [reply src/api/admin.ts]: Oh yeah it will give good hints, but that's a good point also. I like the enum better.

**Pull request merged**